package.json

```diff
@@ -48,6 +48,7 @@
     "babel-preset-stage-1": "^6.3.13",
     "exenv": "^1.2.0",
     "inline-style-prefixer": "^0.6.2",
+    "mobile-detect": "^1.3.1",
```

Comment:
inline-style-prefixer probably already does 90% of this, so we can't afford to pull in a 35k (!) module. 

A better solution might be to always listen for touch events if they are available, and `preventDefault` inside of them so that the other handlers don't fire (the simulated mouse events)